src/server.c

```diff
@@ -2667,6 +2667,7 @@ void initServer(void) {
 
     signal(SIGHUP, SIG_IGN);
     signal(SIGPIPE, SIG_IGN);
+    setupStacktracePipe();
```

Comment:
can't we put this inside setupDebugSigHandlers? 
it's part of the signal handler system (although it also serves assertions).

p.s. maybe we should copy some of the checks we do for watchdog (i.e. matching `bioProcessBackgroundJobs` part of the stack trace), into the assertion test?

Reply:
yes

we can and should